spec/support/config/capybara.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rspec/rails'
+require 'capybara/rspec'
+require 'capybara-screenshot/rspec'
+require 'selenium-webdriver'
+require 'site_prism'
+require 'site_prism/all_there'
```

Comment:
Move it to `site_prism` configuration

Reply:
![image](https://user-images.githubusercontent.com/11984862/114295426-dd6f2300-9aad-11eb-97cf-94352f4ed70c.png)